tests/helpers/utils.py

```diff
@@ -156,9 +156,12 @@ def normalizer_data_from_string(s):
         if action == 'categories':
             category_mappings[curie].extend(line_data)
         elif action == 'synonyms':
-            synset_mappings[curie].extend(line_data)
+            # Add to start of list so that we can override
+            # the primary CURIE
+            synset_mappings[curie][:0] = line_data
```

Comment:
This is prepending? That's crazy.
```suggestion
            synset_mappings[curie] = [line_data] + synset_mappings[curie]
```